packages/serialization/form/kiota_serialization_form/form_parse_node.py

```diff
@@ -219,9 +219,9 @@ def get_enum_value(self, enum_class: K) -> Any:
             if value not in enum_values:
                 raise Exception(f'Invalid value: {value} for enum {enum_class}.')
             result.append(enum_class(value))  # type: ignore
-        return result
+        return result  # type: ignore
 
-    def get_object_value(self, factory: ParsableFactory[U]) -> U:
+    def get_object_value(self, factory: ParsableFactory) -> Parsable:
```

Comment:
This change of the return type will be problematic for auto-completion and type validation of the generated clients I believe.